tsconfig.json

```diff
@@ -6,7 +6,9 @@
     "rootDir": "./src",
     "strict": true,
     "noImplicitAny": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "sourceMap": true,
```

Comment:
The addition of 'sourceMap' is beneficial for debugging, as it allows the original TypeScript code to be mapped back from the compiled JavaScript. This is a good practice for development.